iceoryx2-cli/lib/src/cli.rs

```diff
@@ -12,7 +12,13 @@
 
 use colored::*;
 
-pub fn help_template(cli_name: &str, show_external_commands: bool) -> String {
+pub enum HelpOptions {
+    DontPrintCommandSection,
+    PrintCommandSection,
+    PrintHintForInstalledCommands,
+}
```

Comment:
Food for thought about shorter names

```
pub enum HelpOutput {
    SkipCmdSection,
    ShowCmdSection,
    ShowListCmdHint,
}
```
From the context below, `ShowListCmdHint` could also be `ShowCmdSectionWithListHint` but I'm not sure that's the intention.

Reply:
I prefer the longer names personally. It is also consistent with the code base. I changed the last one as suggested.